apps/teams-test-app/e2e-test-data/webStorage.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Per the recommendation from @KangxuanYe , this file is blank for now. The real tests will be added in a separate change, once the corresponding test file is added in the host SDK repo.

Reply:
Update: Below is the right json file you should follow when you add a new capability / APIs.

{
  "name": "Your CapabilityName",
  "testCases":[{}]
}

Updated here in case someone gets mislead by my words before.